style.css

```diff
@@ -0,0 +1,448 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
```

Comment:
- [ ] Please use the recommended `font-family` from the original design
**Fonts used originally: Cocogoose, Lato**